src/nft-card-back.ts

```diff
@@ -203,11 +235,12 @@ export class NftCardBackTemplate extends LitElement {
         await this.requestUpdate()
       }
     })
-    this.cardHeight = this.shadowRoot.firstElementChild.offsetHeight
-    this.cardWidth = this.shadowRoot.firstElementChild.offsetWidth
+    const el: HTMLElement = this.shadowRoot!.firstElementChild as HTMLElement
+    this.cardHeight = el.offsetHeight
+    this.cardWidth = el.offsetWidth
   }
 
-  public getBoostsTemplate(boosts) {
+  public getBoostsTemplate(boosts: any[]) {
     if (boosts.length <= 0) { return } // Don't render if empty array
```

Comment:
nit: let's use 3 lines for empty return statements to make them easier to catch when scanning code